charts/actions-runner-controller/templates/githubwebhook.ingress.yaml

```
@@ -34,8 +36,15 @@ spec:
           {{- range .paths }}
           - path: {{ .path }}
             backend:
+              {{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
```

You need to scope `Capabilityes` with `$`. Otherwise it is evaluated in the context of `range .paths`, which tries to evaluate something like `$path..Capabilities.APIVersions.Has` which results in `<.Capabilities.APIVersions.Has>: nil pointer evaluating interface {}.APIVersions` error

```suggestion
              {{- if $.Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
```